src/modules/pages/components/ExpenditurePage/LockedSidebar.tsx

```diff
@@ -36,6 +41,25 @@ const LockedSidebar = ({
   const { expenditure, recipients, filteredDomainId, staged } =
     formValues || {};
 
+  const firstFormSection = useMemo(() => {
+    switch (expenditure) {
+      case ExpenditureTypes.Advanced: {
+        return null;
+      }
+      case ExpenditureTypes.Split: {
+        return null;
+      }
+      case ExpenditureTypes.Staged: {
+        return null;
+      }
+      case ExpenditureTypes.Streaming: {
+        return <LockedStreaming startDate={startDate} endDate={endDate} />;
+      }
+      default:
+        return null;
+    }
+  }, [expenditure]);
+
```

Comment:
Are you planning to update this at a later date? If not, it could be simplified to:

``` js
if(expenditure === ExpenditureTypes.Streaming) {
        return <LockedStreaming startDate={startDate} endDate={endDate} />;
      } 
return null
```

Reply:
you're correct, made sure how it changes and replaced the switch with condition because only streaming will have different locked form